pkg/measure-config/build.yml

```diff
@@ -0,0 +1,20 @@
+image: eve-measure-config
+org: lfedge
+
+network: yes
```

Comment:
Can you build this without network? That is the path we are taking to get a reliable SBOM (see #2861 ).

That means yoyu need to run go mod vendor to make sure the vendor directory is populated in github.

Reply:
I've removed networking, I probably should remove extra binds too